src/common/pages/edit/containers/index.js

```diff
@@ -1,17 +1,33 @@
 import { connect } from 'react-redux';
-import Edit from '../components';
-import { onDecrement, onIncrement } from '../store/index';
 
-const mapStateToProps = state => ({
-  counter: state.edit.count,
-});
+import Edit from 'common/pages/edit/components/PasswordForm';
+import addPassword from 'common/api/addPassword';
+import {
+  addPasswordBegin,
+  addPasswordSuccess,
+  addPasswordFailure,
```

Comment:
1. This is editing page, not password creation page.
2. For initial action name use word `Pending`. Like we have pending in `Promises`.